src/graphql/data/nft/CollectionSearch.ts

```diff
@@ -80,9 +79,8 @@ function useCollectionQuerySearch(query: string, skip?: boolean): useCollectionS
 }
 
 export function useCollectionSearch(queryOrAddress: string): useCollectionSearchReturnProps {
-  const isNftGraphqlEnabled = useNftGraphqlEnabled()
   const isName = !isAddress(queryOrAddress.toLowerCase())
-  const queryResult = useCollectionQuerySearch(queryOrAddress, isNftGraphqlEnabled ? !isName : true)
-  const addressResult = useCollection(queryOrAddress, isNftGraphqlEnabled ? isName : true)
+  const queryResult = useCollectionQuerySearch(queryOrAddress, !isName)
+  const addressResult = useCollection(queryOrAddress, isName)
```

Comment:
The second parameter of both of these methods should be changed to an options pack (eg `{ isQuery: boolean }`) - as is, it's very unclear that they're being called correctly.

Reply:
Can you go into more detail here? The second param states whether or not to skip the query and save on a call. The first call should only run if given a name to search for matched results, the second is if given an address which should only return 1 matching collection with the corresponding address

Reply:
My point was just that I can't tell what the second parameter is meant to be from the callsite. Considering it's a boolean, I'd prefer it look like `, /* paramName= */ isName)` or, even more preferable, like `, { paramName: isName })`, for future readers.

Reply:
I would need to completely refactor the arguments for both the `useCollectionQuerySearch` and `useCollection` to allow for the named arguments and these functions are currently used in several locations throughout the app. To avoid scope creep and potentially introducing bugs into this chore PR, I've used your first suggestion